NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -209,10 +209,10 @@ private World CreateFreshWorld()
                                                               NitroxServiceLocator.LocateService<Dictionary<string, List<PrefabAsset>>>());
 
             Log.Info("World GameMode: " + gameMode);
-
-            Log.Info("Server Password: " + (string.IsNullOrEmpty(config.ServerPassword) ? "None. Public Server." : config.ServerPassword));
-            Log.Info("Admin Password: " + config.AdminPassword);
-
+            var serverPass = (string.IsNullOrEmpty(config.ServerPassword) ? "None. Public Server." : config.ServerPassword);
+            Log.DebugSensitive("Server Password: {0} ", serverPass);
```

Comment:
Should be kept as info because in RELEASE mode debug is disabled and then logs will not print.

Reply:
Yup thats a miss on my part there moving from info to debug